actors/miner/src/lib.rs

```diff
@@ -4551,15 +4558,19 @@ fn termination_penalty(
 
     for sector in sectors {
         let sector_power = qa_power_for_sector(sector_size, sector);
+        let mut replaced_sector_age = current_epoch - sector.power_base_epoch;
```

Comment:
FIX: this should be the _replaced_ ("old") sector's age, so power_base - activation.